visualizer/ui.js

```diff
@@ -1,11 +1,14 @@
 'use strict'
 
+const d3 = require('./d3.js')
+const path = require('path')
 const events = require('events')
 const htmlContentTypes = require('./html-content-types.js')
 const debounce = require('lodash.debounce')
 const DataTree = require('./data-tree.js')
 const History = require('./history.js')
 const spinner = require('@nearform/clinic-common/spinner')
+const logo = require(path.join(__dirname, 'nearform-logo.svg'))
```

Comment:
You can only `require()` Node.js scripts and JSON files. To read files, `fs.readFileSync()` is the appropriate function.

`fs` is a Node.js module that is not available in browsers, and this visualizer code only runs in the browser. But the visualizer is built with browserify, and uses the brfs transform:
https://github.com/nearform/node-clinic-common/blob/7e6ed2f74b08bfc60fd690c4c19e0714a04d80d9/scripts/build-js.js#L18
`brfs` can handle some `fs` functions at build time, so you can do
```js
const fs = require('fs')
const logo = fs.readFileSync(path.join(__dirname, 'nearform-logo.svg'), 'utf8')
```
and brfs will replace that `fs.readFileSync()` call with the contents of the svg as a string. I think with that change, this should be well on its way!

Reply:
@goto-bus-stop - Thanks. The svg is being pulled in now. Just working on the d3 code to append the a tag etc.